### PR TITLE
navigation around payment management page

### PR DESCRIPTION
**Pull request by kiranpal-atos**

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -22,4 +22,9 @@
 //
 //
 // -- This is will overwrite an existing command --
-// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
```
Comment (GodsonLeigh) [inline cypress/support/commands.js:25]: should this have a return ?
https://docs.cypress.io/api/cypress-api/custom-commands#Overwrite-visit-command

**Review by GodsonLeigh — COMMENTED** [review]
Loving the use of the cucumber / cypress link up, only one comment relating to the Cypress.Commands.overwrite visit, and the return that says should be specified in the linked docs.

File: cypress/support/commands.js (hunk 1 above)
Comment (kiranpal-atos) [reply cypress/support/commands.js:25]: so these are actually placeholder steps cypress included when you set up a project from scratch, and they comment them out by default - we don't actually need them so i can just delete them

Comment (GodsonLeigh) [reply cypress/support/commands.js]: :) thanks for the clarification

**Review by GodsonLeigh — APPROVED** [review]
:)

**Pull request merged**